hw/opentitan/ot_spi_device.c

```diff
@@ -43,10 +43,10 @@
 #include "hw/riscv/ibex_irq.h"
 #include "trace.h"
 
-#define PARAM_SRAM_DEPTH         1024u
-#define PARAM_SRAM_OFFSET        4096u
-#define PARAM_SRAM_EGRESS_DEPTH  832u
-#define PARAM_SRAM_INGRESS_DEPTH 104u
+#define PARAM_SRAM_OFFSET        0x1000u
+#define PARAM_SRAM_DEPTH         (0x1000u / sizeof(uint32_t))
+#define PARAM_SRAM_EGRESS_DEPTH  (0xe00u / sizeof(uint32_t))
```

Comment:
I think this does not match the size in the [docs](https://opentitan.org/earlgrey_1.0.0/book/hw/ip/spi_device/doc/registers.html#egress_buffer). Should this instead be:

```suggestion
#define PARAM_SRAM_EGRESS_DEPTH  (0xd40 / sizeof(uint32_t))
```

Because I think we would want to error if somebody wrote to an offset of `0x1d50`, for example.